### PR TITLE
docs(BPMN): adds documentation for boundary events

### LINKED ISSUE #1722: Document boundary events
- add short blurb about boundary event in BPMN primer
- update BPMN coverage table
- add boundary event section under BPMN Workflows

Not sure if it makes sense to update the modeler section.

### PR DESCRIPTION
**Pull request by npepinpe**
- expands documentation on events in BPMN
- updates timer documentation to reflect changes in how durations are
parsed
- updates timer documentation to add timerWithCycle
- updates bpmn coverage table to show interrupting message/timer
boundary events as implemented

closes #1722

### CODE REVIEW

**Comment by npepinpe** [comment]
One thing to note, I'm not 100% sure about the added documentation in the events main page. Also the image kind of sucks, and it seemed a good idea at the time but now I'm not so sure...

File: docs/src/bpmn-workflows/events.md [hunk 1]
```diff
@@ -1,7 +1,49 @@
...
+* Intermediate events can be used to indicate that something has happened
+(i.e. intermediate throw events), or to wait and react to certain events
+(i.e. intermediate catch events).
+
+## Intermediate events
+
+Intermediate catch events can be inserted into your process in two different
+contexts: normal flow, or attached to an activity, and are called boundary events.
+
+### Normal flow
```
Comment (saig0) [inline docs/src/bpmn-workflows/events.md]: I would not say that a boundary event is an intermediate event that is attached to an activity. I think this doesn't fit to BPMN well. 

```suggestion
## Intermediate Events
```

**Review by saig0 — CHANGES_REQUESTED** [review]

File: docs/src/bpmn-workflows/timer-events.md [hunk 2]
```diff
@@ -10,10 +10,13 @@ A timer intermediate event acts as a stopwatch. When a token arrives at the time
 
 The duration must be defined in the [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601#Durations) format. For example:
 
+* `P1Y2M3DT1H2M3S` - 1 year, two months, 3 days, 1 hour, 2 minutes, and 1 second.
 * `PT15S` - 15 seconds
 * `PT1H30M` - 1 hour and 30 minutes
 * `P14D` - 14 days
 
+Durations can also be negative; a negative timer will fire immediately, and can be expressed as `-PT5S`, or minus 5 seconds.
```
[the remark this answers is not in the document]
Comment (npepinpe) [reply docs/src/bpmn-workflows/timer-events.md]: How did you do that? :open_mouth:

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: c0b239523bf725a79cab19dadf133c3e44b56c58
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

Guidelines are available at https://github.com/zeebe-io/zeebe/blob/develop/CONTRIBUTING.md

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: c0b239523bf725a79cab19dadf133c3e44b56c58
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

Guidelines are available at https://github.com/zeebe-io/zeebe/blob/develop/CONTRIBUTING.md

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: c0b239523bf725a79cab19dadf133c3e44b56c58
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

 * Commit: ac9626c677fd1bca1c37a7a7272eb8fe7d89edbf
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

Guidelines are available at https://github.com/zeebe-io/zeebe/blob/develop/CONTRIBUTING.md

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: c0b239523bf725a79cab19dadf133c3e44b56c58
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

 * Commit: ac9626c677fd1bca1c37a7a7272eb8fe7d89edbf
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

 * Commit: 18fcb258c750618135b1c31c15ca11db20bc9709
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

Guidelines are available at https://github.com/zeebe-io/zeebe/blob/develop/CONTRIBUTING.md

---
This message was auto-generated by https://gitcop.com

File: docs/src/bpmn-workflows/events.md [hunk 3]
```diff
@@ -1,7 +1,49 @@
 # Events
 
-Currently supported elements:
+![workflow](/bpmn-workflows/events.png)
```
[the remark this answers is not in the document]
Comment (npepinpe) [reply docs/src/bpmn-workflows/events.md]: Yeah, like I said in my comment, it seemed a good idea at the time, but then I wasn't so sure, and I figured a second opinion would be interesting. I think I would not even have the image at all.

**Comment by npepinpe** [comment]
I think I addressed all feedback points with the last commits.

**Review by saig0 — APPROVED** [review]

**Pull request merged**